src/main/java/org/broadinstitute/consent/http/resources/DatasetResource.java

```diff
@@ -370,6 +371,23 @@ public Response getDataset(@PathParam("datasetId") Integer datasetId) {
     }
   }
 
+  @GET
+  @Path("/study/{studyId}")
+  @Consumes("application/json")
```

Comment:
Remove this line since we're not consuming any json. This bug took quite a while to debug. Hitting the endpoint from swagger, this always returned a 404 that was served from the root dropwizard application due to the consumes header mismatch.